### PR TITLE
Fixed errors introduced to the XR Rig. Cleaned up some scene overrides. Added missing license header

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
The MRTK Rig had an error introduced into it where the near mode detector had a null object set as a managed controller. This PR also cleans up several scenes which had leftover CanvasProxyInteractors in the scene. The CanvasProxyInteractor is now automatically included in the MRTK rig.

This PR also fixes some other instances in the examples where there were unnecessary overrides placed on the base MRTK rig.

It also adds a missing license header to a class.

## Changes
- Fixes: #10882

### CODE REVIEW

**Comment by Zee2** [comment]
Will multiple CanvasProxyInteractors break anything now? We should be resilient to that kind of misconfiguration

**Comment by RogPodge** [comment]
> Will multiple CanvasProxyInteractors break anything now? We should be resilient to that kind of misconfiguration

We don't think it was the root cause of the issue upon further analysis. They've been removed from the scenes they were present so as to prevent accidentally clicking canvas MRTK Interactables while using input sim.

**Review by Clandguth — APPROVED** [review]
Nothing looks out of place. Will run another manual pass once this PR is merged. Will still need a review from a reviewer with write access

**Review by Zee2 — APPROVED** [review]

**Pull request merged**
